types/schemas/prototypeApplication/data/shared.ts

```diff
@@ -2,7 +2,7 @@ import {URL} from '../../../shared/utils';
 import {UKResidentialUnitType} from '../enums/ResidentialUnitType';
 import {UKTenureType} from '../enums/TenureType';
 
-export type Entity = {
+export type PrototypeEntity = {
```

Comment:
This re-naming fixes a duplicate type export error when running `pnpm build-schema`

Reply:
Yep this was one of the build issues I also hit!